corehq/apps/accounting/tasks.py

```diff
@@ -787,36 +836,64 @@ def _downgrade_domain(subscription):
 
 
 def _send_downgrade_warning(invoice, context):
+    if invoice.is_customer_invoice:
+        subject = _(
+            "CommCare Alert: {}'s subscriptions will be downgraded to Community Plan after tomorrow!".format(
+                invoice.account.name
+            ))
+        subscriptions_to_downgrade = _(
+            "subscriptions on {}".format(invoice.account.name)
+        )
+        bcc = None
+    else:
+        subject = _(
+            "CommCare Alert: {}'s subscription will be downgraded to Community Plan after tomorrow!".format(
+                invoice.get_domain()
+            ))
+        subscriptions_to_downgrade = _(
+            "subscription for {}".format(invoice.get_domain())
+        )
+        bcc = [settings.GROWTH_EMAIL]
+
+    context.update({
+        'subscriptions_to_downgrade': subscriptions_to_downgrade
+    })
     send_html_email_async.delay(
-        _("CommCare Alert: {}'s subscription will be downgraded to Community Plan after tomorrow!".format(
-            invoice.get_domain()
-        )),
+        subject,
         invoice.contact_emails,
         render_to_string('accounting/email/downgrade_warning.html', context),
         render_to_string('accounting/email/downgrade_warning.txt', context),
         cc=[settings.ACCOUNTS_EMAIL],
-        bcc=[settings.GROWTH_EMAIL],
+        bcc=bcc,
         email_from=get_dimagi_from_email())
 
 
 def _send_overdue_notice(invoice, context):
+    if invoice.is_customer_invoice:
+        bcc = None
+    else:
+        bcc = [settings.GROWTH_EMAIL]
     send_html_email_async.delay(
-        _('CommCare Billing Statement 30 days Overdue for {}'.format(invoice.get_domain())),
+        _('CommCare Billing Statement 30 days Overdue for {}'.format(context['domain_or_account'])),
         invoice.contact_emails,
         render_to_string('accounting/email/30_days.html', context),
         render_to_string('accounting/email/30_days.txt', context),
         cc=[settings.ACCOUNTS_EMAIL],
-        bcc=[settings.GROWTH_EMAIL],
+        bcc=bcc,
         email_from=get_dimagi_from_email())
 
 
 def _create_overdue_notification(invoice, context):
+    if invoice.is_customer_invoice:
+        domains = [subscription.subscriber.domain for subscription in invoice.subscriptions.all()]
+    else:
+        domains = [invoice.get_domain()]
     message = _('Reminder - your {} statement is past due!'.format(
         invoice.date_start.strftime('%B')
     ))
     note = Notification.objects.create(content=message, url=context['statements_url'],
-                                       domain_specific=True, type='billing',
-                                       domains=[invoice.get_domain()])
+                                       domain_specific=False if invoice.is_customer_invoice else True,
```

Comment:
What does ```domain_specific``` do?

Reply:
@nickpell  I'm not sure, but I figured it should remain True for regular invoices, but be False for customer invoices, since customer invoices are not tied to a single domain. I could be wrong about how `domain_specific` is interpreted, though.

Reply:
My interpretation of ```Notification.get_by_user``` is that if ```domain_specific == False```, then the notification will be visible for all users, which isn't the intent here.

Reply:
Makes a lot of sense. I'll change it back so `Notification.domain_specific` is False for all invoices